knotx-example/knotx-example-monolith/src/test/java/com/cognifide/knotx/example/monolith/SampleApplicationHeadersTest.java

```diff
@@ -55,14 +60,15 @@ public void before() {
   }
 
   @Test
+  @KnotxConfiguration("knotx-example-monolith.json")
   public void whenRequestingRemoteRepository_expectOnlyAllowedResponseHeaders(TestContext context) {
     testGetRequest(context, REMOTE_REQUEST_URI);
   }
 
   private void testGetRequest(TestContext context, String url) {
```

Comment:
`expectedHeaders` could also be argument for this method. (Just saying, this could be also done later).